docker/images/outrigger/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM ubuntu:14.04
```

Comment:
Rather than installing node by hand below, seems like we should base this image off one of the official node images: https://hub.docker.com/_/node/